src/main/java/com/hladchenko/springrestapi/entity/User.java

```diff
@@ -0,0 +1,19 @@
+package com.hladchenko.springrestapi.entity;
+
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@Data
+public class User {
+
+    private UUID id;
+    private String name;
+    private List<Note> notes = new ArrayList<>();
+
+    public void addNote(Note note) {
+        notes.add(note);
```

Comment:
If you expect somebody to use your `notes` list only in this way, then disable `notes` field from being set by other collection and if somebody will need to get it by getter method, provide read-only view on that collection.